### PR TITLE
[signing] Publish Signing Ceremony 2024-05-24

### PR DESCRIPTION
**Pull request by moidx**
A signing ceremony was performed to sign the following items:
- The ProdA, ProdC and Sival ROM_EXTs.
- Personalization binaries.

The ROM_EXT now reports version 0.2 secver: 1

### CODE REVIEW

File: sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_silicon_creator.signed.bin
Comment (timothytrippel) [inline sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_silicon_creator.signed.bin:1]: I think you need to manually change these file names to match what is there: https://cs.opensource.google/opentitan/opentitan/+/earlgrey_es_sival:sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/BUILD;drc=f67fc1722a98421311592d1f2e4062cd0753c0a9;l=24

(Applies for all perso bins)

File: sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_fpga_cw310_rom_with_fake_keys.signed.bin
Comment (timothytrippel) [inline sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_fpga_cw310_rom_with_fake_keys.signed.bin:1]: we don't currently check in the FPGA perso bins: https://cs.opensource.google/opentitan/opentitan/+/earlgrey_es_sival:sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/

We seem to check in the FPGA signatures though which seems like a mistake.

File: sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_silicon_creator.signed.bin
Comment (moidx) [reply sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_silicon_creator.signed.bin:1]: Fixed.

File: sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_fpga_cw310_rom_with_fake_keys.signed.bin
Comment (moidx) [reply sw/device/silicon_creator/manuf/skus/earlgrey_a0/sival_bringup/binaries/ft_personalize_1_fpga_cw310_rom_with_fake_keys.signed.bin:1]: Fixed.

**Review by cfrantz — APPROVED** [review]

**Review by timothytrippel — APPROVED** [review]

**Pull request merged**